scipy/signal/tests/test_windows.py

```diff
@@ -707,6 +707,24 @@ def test_needs_params():
         assert_raises(ValueError, get_window, winstr, 7)
 
 
+def test_not_needs_params():
+    for winstr in ['barthann',
+                   'bartlett',
+                   'blackman',
+                   'blackmanharris',
+                   'bohman',
+                   'boxcar',
+                   'cosine',
+                   'flattop',
+                   'hamming',
+                   'hanning',
+                   'nuttall',
+                   'parzen',
+                   'taylor',
+                   'triangle']:
+        get_window(winstr, 7)  # should work without error
```

Comment:
While we are here, can we also check the result has 7 items to make sure the argument is actually passed? 

```python
win = get_window(winstr, 7)
assert_equal(win.len, 7)
```

Reply:
I agree. I fixed.